stories/Radio.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { ComponentMeta } from "@storybook/react"
+
+import { Radio, RadioGroup as _RadioGroup } from "../src/index"
+import { FormControlLabel } from "@mui/material"
+
+export default {
+  title: "Example/Radio",
+  component: Radio,
+} as ComponentMeta<typeof Radio>
+
+export const RadioGroup = () => (
```

Comment:
you can avoid the `_` trick by using capitalization:

```suggestion
import { Radio, RadioGroup } from "../src/index"
import { FormControlLabel } from "@mui/material"

export default {
  title: "Example/Radio",
  component: Radio,
} as ComponentMeta<typeof Radio>

export const radioGroup = () => (
```